### DISCUSSION TITLE
12/29

### DISCUSSION THREAD

**Discussion by SoobinJ** [Team-Plants/PLANTS#91] [Daily Scrum]
## 김윤후

- 오늘의 컨디션 : 2점.. 술을 마셨어요..
- 어제 진행했거나 완료한 작업 : 대시보드 수정 기능 개발중~
- 오늘 진행할 작업 : 대시보드 수정 기능 완료 예정
- 막혔던 부분 : x
- 도움이 필요한 부분 : x
- TMI : 충무로 하얀집..? 복분자주.. 와우..막차시간까지 마셔마셔

## 송민혁

- 오늘의 컨디션 : 7점
- 어제 진행했거나 완료한 작업 : 칼럼 추가 하는 중
- 오늘 진행할 작업 : 마무리 할 예정, 내일까지 해야 하는 것도 오늘 다 할 예정
- 막혔던 부분 : 무한스크롤.. (공부하는 중이라 좀 이따 질문할게요!)
- 도움이 필요한 부분 : x
- TMI : 발표 생각하니까 발표 구상중(미친 사람처럼 하고싶다고 함...ㅎㅎ)

## 안혜정

- 오늘의 컨디션 : 5점
- 어제 진행했거나 완료한 작업 : 할일 추가 모달 기능 구현
- 오늘 진행할 작업 : 이미지 url api 수정 예정, 대시보드  예정 
- 막혔던 부분 : x
- 도움이 필요한 부분 : x
- TMI : 밤을..ㅠㅠ 하지만 편안 상태~ 끝나고 잘 예정

## 전수빈

- 오늘의 컨디션 : 5점
- 어제 진행했거나 완료한 작업 : redirect, 초대하기 기능 개발
- 오늘 진행할 작업 : 초대하기 기능 마무리 
- 막혔던 부분 : redirect....
- 도움이 필요한 부분 :
- TMI : 데블스 플랜 꿀잼.. 새벽3시까지 봄...추천!

## 한슬희

- 오늘의 컨디션 : 8점
- 어제 진행했거나 완료한 작업 : 할일 카드 모달 - 정보, 댓글 가져오기 등등 완료
- 오늘 진행할 작업 : 나머지 마무리할 예정
- 막혔던 부분 : x
- 도움이 필요한 부분 : x
- TMI : 하늘님이 우리 레포 눌러줌

**Comment by hanseulhee** [comment]
ㅋㅋㅋㅋㅋ 말투 귀여워요
